playwright.config.ts

```diff
@@ -0,0 +1,32 @@
+import { devices, PlaywrightTestConfig } from '@playwright/test';
+
+// Config file docs: https://playwright.dev/docs/test-configuration
+const config: PlaywrightTestConfig = {
+  webServer: {
+    command: 'pnpm start',
+    port: 3000,
+    reuseExistingServer: !process.env.CI,
+  },
+  testIgnore: '**/util/__tests__/**',
+  retries: process.env.CI ? 2 : 0,
```

Comment:
The maximum number of retry attempts given to failed tests. The default value is `0` 
https://playwright.dev/docs/api/class-testconfig#test-config-retries

Reply:
let's remove it and see what happens 👍  if we need to add it back, we always can do that in the future (with a justification)

Reply:
removed in this commit
[10f606a](https://github.com/upleveled/next-js-portfolio-web/pull/107/commits/10f606a1ad3d180595e6886d7f020fca3cd484fd)